### DISCUSSION TITLE
The harvester cannot be connected to the host, but the host wallet can display the connected harvester

### DISCUSSION THREAD

**Discussion by KmBooy** [Chia-Network/chia-blockchain#6331] [Support]
The harvester cannot be connected to the host, but the host wallet can display the connected harvester.I don't know what caused this, but he has kept me from exploding for a long time.Please help me who knows
![image](https://user-images.githubusercontent.com/35790216/120409693-0f9d5280-c384-11eb-9dd2-645705535052.png)
![image](https://user-images.githubusercontent.com/35790216/120409805-50956700-c384-11eb-8cc9-1f047b830cde.png)
![image](https://user-images.githubusercontent.com/35790216/120409844-64d96400-c384-11eb-9154-271ad15da705.png)

**Comment by BrandtH22** [comment]
@KmBooy , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
